tests/test_serialize_sequence.cc

```diff
@@ -30,6 +30,7 @@
 #include <sstream>
 #include <system_error>
 #include <vector>
+#include <fstream>
```

Comment:
In other places we order the includes alphabetically (one block for C++ headers, one for system headers, one for our one headers):
```
#include <algorithm>
#include <string>

#include <gul14/gul.h>
#include <unistd.h>

#include "my_own_header.h"
```

Reply:
Ah, ok. I thought there is no guidance about header includes (in our C++ Style Guide from DESY I don't find any header ordering). In the past I try to separate them but unordered. I will move the `fstream` up.

Reply:
If there are many headers it's easier to find if some specific header is already there.
And usually if you have lists of 'unsorted' things, like file names in `meson.build` files, just sort them to facilitate easier handling.